OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -15,6 +15,8 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum ReferencePoints { TopLeft, BottomRight }
```

Comment:
These options would be useful for a future PR that could use the same positioning options for pips (default: BottomLeft) and tags (default: TopCenter) too.